.circleci/config.yml

```diff
@@ -236,6 +236,7 @@ workflows:
             branches:
               ignore: /.*/
       - deploy_studiopress:
+          context: wpe-product-info-aws
```

Comment:
@BMO-tech should this be `context: studiopress-deploy`?

Reply:
Yes, sorry. Good catch